src/templates/weekEnded/html.mjml

```diff
@@ -1,41 +1,85 @@
-<mjml>
+<mjml owa="desktop" version="4.6.3">
   <mj-head>
     <mj-title>Week {{ week }} has just finished</mj-title>
-    <mj-preview>{{{formatPreview 'This is an automated email you requested to let you know when the week ends'}}}</mj-preview>
+    <mj-preview>{{{'This is an automated email you requested to let you know when the week ends'}}}</mj-preview>
```

Comment:
@balexander82 why was this taken out?  We have to have the formatPreview function here for the preview to work